client/utils.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import (
+	"math/big"
+	"crypto/rand"
+)
+
+func getRandom(q *big.Int) *big.Int {
+	x, err := rand.Int(rand.Reader, q)
+	if err != nil {
+        panic(err)
+    }
+	return x
+}
+
+func isVeto(listOfVotes []*big.Int, sizeOfList int, p *big.Int) bool {
```

Comment:
`isVeto` should be a private method of `FurtiveClient`, since it provides context-specific logic not useful outside of that receiver.

Move it to client and
```suggestion
func (fc *FurtiveClient) isVeto(listOfVotes []*big.Int, sizeOfList int, p *big.Int) bool {
```

Reply:
And also, something is not yes:
```
pawelnarolski@DESKTOP-O8I5MSG client % go run *.go
Question: Have you ever shoplifted?
Do you want to veto? (Y/N): Y
No veto
```

Reply:
> And also, something is not yes:
> 
> ```
> pawelnarolski@DESKTOP-O8I5MSG client % go run *.go
> Question: Have you ever shoplifted?
> Do you want to veto? (Y/N): Y
> No veto
> ```

It's because there is only one player now